.devops/pagopa-deploy-pipelines.yml

```diff
@@ -71,9 +71,10 @@ stages:
 
           - template: azure-templates/setup-env.yml
             parameters:
-              api_payment_methods_basepath: '/payment-methods/checkout/v1'
-              pagopa_wallet_cdn: 'https://assets.cdn.platform.pagopa.it/payment-methods'
-              pagopa_logos_cdn: 'https://assets.cdn.io.italia.it/logos/abi'
+              api_host: 'https://api.dev.platform.pagopa.it'
+              api_wallet_basepath: '/webview-payment-wallet/v1'
+              api_pm_basepath: '/pp-restapi-CD'
+              api_timeout:  10000
```

Comment:
this approach can only work with `env-config.js`

Reply:
what do you think about adding `env-config.js` to the index.html and then evaluating a possible refactoring in the next sprint? @stratoivandiluccio

Reply:
adding now the env-config.js will not resolve the issue, because I am using the env variables at compilation time and not at run time unfortunately